app/views/spree/orders/_totals_footer.html.haml

```diff
@@ -23,6 +23,22 @@
           = t :order_total_price
       %td.text-right.total
         %h5#order_total= order.display_total.to_html
+    %tr.total
+      %td.text-right{colspan: "3"}
+        %h5
+          = t :order_amount_paid
+      %td.text-right.total
+        %h5
```

Comment:
I'm not sure if a certain style was specified, but I think the `order_amount_paid` could be formatted smaller, the same as the `order_produce` row. What do you think?

```suggestion
        %strong
          = t :order_amount_paid
      %td.text-right.total
        %strong
```

Reply:
Changed tag from h5 to strong.